src/matching.js

```diff
@@ -331,21 +331,22 @@ function findMaximumMatching(n) {
 }
 
 function buildGraph(matches, people) {
-  let graph = [];
-  let count = Object.keys(people).length + 1;
+  let count = people.length;
   let edges = 0;
-  for (var i = 1; i < count; i++) {
+  for (var i = 0; i < count; i++) {
     for (var j = i + 1; j < count; j++) {
       if (!people[i].added || !people[j].added) continue;
+      const IId = people[i].id;
+      const JId = people[j].id;
```

Comment:
convention is to use low level case right? so maybe:
```suggestion
      const iID = people[i].id;
      const jID = people[j].id;
```